instagram/app/assets/stylesheets/photos.scss

```diff
@@ -0,0 +1,138 @@
+// Place all the styles related to the photos controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+h2 {
+	font-size: 16px;
+}
+
+#post {
+	border: 1px solid #dbc7c7;
+	margin-bottom: 10px;
+	padding-bottom: 5px;
+	background: white;
+}
+
+#postHeader{
+	background-color: white;
+	padding: 0px 10px;
+}
+
+
+#userName {
+	display: inline-block;
+	width: 50%;
+	color: #000080;
+	font-size: 80%;
+}
+
+#createdAt {
+	float:right;
+	display: inline-block;
+	color: #000080;
+	font-size: 80%;
+}
+
+#photo {
+	width:100%;
+}
+
+#postFooter {
+}
+
+#likeCounter {
+	display: inline-block;
+	padding: 5px 10px 0px 10px;
+}
+
+#likeButton {
+	float: right;
+
+}
+
+#new_like {
+	display: inline-block;
+	float: right;
+	padding: 5px 5px 0 10px;
+}
+
+#userNameDescription {
+	padding: 0px 5px 0 10px;
+	font-weight: 600;
+	color: #000080;
+}
+
+#photoDescription {
+	padding-left:10px;
+}
+
+#addComment {
+	width: 75%;
+	height: 30px;
+	display: inline-block;
+	padding: 0 1%;
+	margin-left: 1%;
+	margin-top: 2%;
+}
+
+#addCommentSubmit {
+	display:inline-block;
+	border: 1px solid #dbc7c7;
+	background: #dbc7c7;
+	opacity: 0.6;
+	height: 32px;
+	width: 19%;
+	margin-right: 1%;
+}
+
+#comments {
+	border-top: 1px solid #dbc7c7;
+	padding: 0px 10px;
+}
+
+#comments ul {
+	list-style-type: none;
+	padding-left: 0px;
+}
+
+#commentUsername {
+	padding-right: 5px;
+	color: #000080;
+	font-weight: 600;
+}
+
+
+#deletePhoto {
+	padding: 5px 10px 0 10px;
+	display: inline-block;
+	float: right;
+}
+
+#commentPhoto {
+		padding: 5px 10px 0 10px;
```

Comment:
Line should be indented with spaces, not tabs<br>Shorthand form for property `padding` should be written more concisely as `5px 10px 0` instead of `5px 10px 0 10px`